pandas/core/window/rolling.py

```diff
@@ -436,6 +437,16 @@ def hfunc2d(values: ArrayLike) -> ArrayLike:
             new_mgr = mgr.apply_2d(hfunc2d, ignore_failures=True)
         else:
             new_mgr = mgr.apply(hfunc, ignore_failures=True)
+
+        if 0 != len(new_mgr.items) != len(mgr.items):
+            # GH#42738 ignore_failures dropped nuisance columns
+            warnings.warn(
+                "Dropping of nuisance columns in rolling operations "
```

Comment:
can you list the columns that are problematic in the error message?

Reply:
updated + green